src/entity/index.ts

```diff
@@ -27,6 +27,12 @@ export class IasqlDatabase {
   })
   recordCount: number;
 
+  @Column({
+    type: 'int',
+    default: 0,
+  })
+  operationCount: number;
```

Comment:
Maybe something like this could also work and reduce some code? https://www.postgresql.org/docs/current/ddl-generated-columns.html
Not sure if can be used cross tables.

Reply:
oh that would be nice! unfortunately it only works for columns within the same row